pkg/api/controllers/account_controller.go

```diff
@@ -19,6 +20,29 @@ func NewAccountController() AccountController {
 	return AccountController{}
 }
 
+type BalanceOperator string
+
+const (
+	BALANCE_OPERATOR_E   BalanceOperator = "e"
```

Comment:
Usually, constants in go are in CamelCase.

Reply:
Oh right ! i'm fixing that